storage/wdpost/wdpost_run_test.go

```diff
@@ -298,7 +298,7 @@ func TestWDPostDoPostPartLimitConfig(t *testing.T) {
 	//stm: @CHAIN_SYNCER_COLLECT_CHAIN_001, @CHAIN_SYNCER_COLLECT_HEADERS_001, @CHAIN_SYNCER_VALIDATE_TIPSET_001
 	//stm: @CHAIN_SYNCER_NEW_PEER_HEAD_001, @CHAIN_SYNCER_VALIDATE_MESSAGE_META_001, @CHAIN_SYNCER_STOP_001
 	ctx := context.Background()
-	expectedMsgCount := 364
+	expectedMsgCount := 8
```

Comment:
@magik6k Please review changes to magic numbers here.

Reply:
Those seem to make sense